src/CMakeLists.txt

```diff
@@ -153,23 +156,23 @@ set(edb_INCLUDES
 	${PROJECT_SOURCE_DIR}/include/QLongValidator.h
 	${PROJECT_SOURCE_DIR}/include/QULongValidator.h
 	${PROJECT_SOURCE_DIR}/include/HexStringValidator.h
-	${PROJECT_SOURCE_DIR}/include/Instruction.h		
```

Comment:
Such whitespace changes shouldn't go into the commit if they are unrelated to commit's objective: they distract the reader from the "useful" changes.

Reply:
Sorry, this wasn't my intention